packages/nitro-protocol/package.json

```diff
@@ -71,6 +72,7 @@
     "{src,test}/**/*.{ts,tsx}": "eslint"
   },
   "main": "lib/src/index.js",
+  "private": false,
```

Comment:
Why private?

Reply:
!private

Reply:
Oh I see. Seems odd to _add_ this key. Usually it's just omitted